packages/select/README.md

```diff
@@ -0,0 +1,33 @@
+# hixme-ui select
+
+```bash
+npm i --save @hixme-ui/select
+```
+
+## Usage
+
+```javascript
+import Select from '@hixme-ui/select'
+
+<Select> {console.log()}}>
+  <option value='hello'>Hello</option>
+  <option value='goodbye'>Goodbye</option>
+</Select>
+
+
+<Select list={[{
+  id: 0,
+  name: 'Zero'
+}]} />
+// Outputs
+<select>
+  <option value='0'>Zero</option>
+</select>
+```
+
+## Props
+
+| Name            | Type        |
+| --------------- | ----------- |
+| list            | array       |
```

Comment:
One thing I've been doing on these tables is adding a default props column. I think communicating whether whether it's `isRequired` is helpful.

Reply:
cool, that sounds good